tests/evtimer_msg_mbox/tests/01-run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2020 Simon Brummer <simon.brummer@posteo.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    try:
+        child.expect('Test successful')
+        print('Test successful')
+    except Exception:
+        print('Test failed')
```

Comment:
This ironically makes the test succeed (from the perspective of the CI), because there is no error exit case now. If you want output, just make the `echo` parameter of the `run()` function `True` and remove the prints and exception handling.

Reply:
I removed the exception handling